pennylane/ops/op_math/controlled.py

```diff
@@ -864,9 +864,13 @@ def _decompose_custom_ops(op: Controlled) -> list["operation.Operator"]:
         # has some special case handling of its own for further decomposition
         return _decompose_pauli_x_based_no_control_values(op)
 
-    if isinstance(op.base, qml.GlobalPhase) and len(op.control_wires) == 1:
+    if isinstance(op.base, qml.GlobalPhase):
```

Comment:
There's some custom handling for multi-controlled phase shift below this branch that can theoretically be removed now that multi-controlled global phase is supported.

Reply:
Ah, yeah I was not sure about that, but I can try and remove it :)